assets/js/modules/adsense/datastore/accounts.js

```diff
@@ -1,5 +1,5 @@
 /**
- * modules/adsense data store: accounts.
+ * Adsense module data store: accounts.
```

Comment:
See above, let's keep the store names in here for all module stores.

Reply:
This change wasn't reverted yet for all module stores (we should keep using `modules/adsense` here, and similarly in the other module store files.